### ISSUE TITLE
Count for guest network not matching with Google Wifi/Home App / shows devices even if no one is connected.

### ISSUE THREAD

**Issue by shogun160** [djtimca/hagooglewifi#8]
The count sensors for guest network is showing a different amount of devices than the Google Wifi and Google Home App.
Sensor for Guest Network shows value between  2 & 10 even if no one is connected to the guest Wifi. This also happens when the guest wifi is turned off.
![Screenshot_20201201-120819](https://user-images.githubusercontent.com/63370033/100736836-9b352700-33d3-11eb-9641-88b256392f79.png)
![Screenshot_20201201-120649](https://user-images.githubusercontent.com/63370033/100736843-9cfeea80-33d3-11eb-9c5f-e44695d511e2.png)
![Screenshot_20201201-124236](https://user-images.githubusercontent.com/63370033/100736846-9d978100-33d3-11eb-8656-448b0f60f470.png)

**Comment by djtimca** [comment]
I am trying to detect guest network based on the IP address range of the devices. If they are on the primary IP range they are part of the main network otherwise they are guest. Unfortunately there doesn't appear to be another proxy in the data for what is connected to guest vs primary network.

**Comment by shogun160** [comment]
Thank you for your quick reply and sorting things out for me. Unfortunately I didn't find any option for guest wifi IP range (Apps/instruction manual). How should I set everything up, so that sensor shows the correct value?

**Comment by djtimca** [comment]
I don't think it is something you set. For example my main network is 192.168.86.x and the guest network is set up as 192.168.87.x. BUT that isn't totally consistent. I have devices that pop up onto the 87 subnet occassionally...

I'm trying to find a more reliable/consistent way to pull the guest network count from the APIs. Will keep you posted.

**Comment by albinmedoc** [comment]
Any updates on this?

Would be great to active guest mode when there are any devices connected to the guest network. But this isn't reliable right now.
